data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaSourceCustomConsumer.java

```diff
@@ -238,6 +232,21 @@ void processAcknowledgedOffsets() {
         acknowledgedOffsets.clear();
     }
 
+    private void updateCommitCount(final TopicPartition topicPartition, final OffsetAndMetadata offsetAndMetadata) {
```

Comment:
It appears this only updates the metric, but the name implies it updates an actual count. Perhaps rename to `updateCommitCountMetric`.